src/css/flags/audioplayer.less

```diff
@@ -43,3 +43,9 @@
 .jwplayer.jw-flag-audio-player .jw-preview {
     display: none;
 }
+
+.jwplayer.jw-flag-media-audio.jw-flag-audio-player {
```

Comment:
Yeah, I just tested locally with .jwplayer.jw-flag-audio-player by itself and it didn't work, but with .jwplayer.jw-flag-media-audio.jw-flag-audio-player { it works fine.